src/Chapter1.hs

```diff
@@ -520,7 +521,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if x == (min (abs x) (abs y)) then x else if y == (min (abs x) (abs y)) then y else if (abs x) == (min (abs x) (abs y)) then x else y
```

Comment:
That looks like a valid implementation 👍🏻 

But I think you can implement it shorter by using the `<` or `>` comparison operators and the `abs` function

Reply:
You are right, I had a redundant condition there, just removed